docusaurus/docs/learn/quickstarts/network/local-no-docker.md

```diff
@@ -66,11 +66,12 @@ it up again. You can get the environment variables by sourcing the ".env" file.
 for details)
 
 You can see what your value is set to by running 
-`echo "${ZITI_CTRL_EDGE_ADVERTISED_ADDRESS}:${ZITI_CTRL_EDGE_ADVERTISED_PORT}"`. This value defaults to: 
-`$(hostname -s):1280`. Make sure the controller is on and listening and then start the edge router. 
+`echo ${ZITI_CTRL_EDGE_ADVERTISED_ADDRESS}:${ZITI_CTRL_EDGE_ADVERTISED_PORT}`. This value defaults to: 
+`$(hostname -s):1280`. Make sure the controller by trying to curl to the address and then start the edge router. 
 
 ```bash
-echo "${ZITI_CTRL_EDGE_ADVERTISED_ADDRESS}:${ZITI_CTRL_EDGE_ADVERTISED_PORT}"
+curl "${ZITI_CTRL_EDGE_ADVERTISED_ADDRESS}:${ZITI_CTRL_EDGE_ADVERTISED_PORT}"

```

Comment:
i refuse to add the "$"

Reply:
a logical reason for not having a `$` here (or probably any prompt for that matter) is that rendered block quotes include a "copy" control, and pasting a command with a leading `$` will lead to confusing results if blindly pasted.
<img width="1028" alt="image" src="https://github.com/openziti/ziti-doc/assets/13525299/fef27d62-25bc-4471-bbd3-0b40405b704f">

Reply:
separated the output

Reply:
The convention I like is not to include the prompt `$` when intended for copying, i.e., when no output is shown. I typically do include the prompt `$` when no output is provided because it is intended for copy/paste.

Reply:
every prompt is different, mac for example with zsh shows %... I much prefer to not show the $ prompt at all and just show the actual output.